flow/test/test_helper.sh

```diff
@@ -63,4 +63,34 @@ if [ ! -z ${MAKE_ISSUE+x} ]; then
   $__make final_report_issue 2>&1 | tee -a $LOG_FILE
 fi
 
-exit $ret
+# Run Autotuner CI specifically for gcd/selected platforms.
+RUN_AUTOTUNER=0
+case $DESIGN_NAME in
+  "gcd")
+    RUN_AUTOTUNER=1
+    ;;
+esac
+if [ $RUN_AUTOTUNER -eq 1 ]; then
+  case $PLATFORM in
+       "gf180" | "nangate45" | "sky130hd_fakestack" | "sky130hs")
+        RUN_AUTOTUNER=0
+        ;;
+  esac
+fi
+
+if [ $RUN_AUTOTUNER -eq 1 ]; then
+  # change directory to ../
+  cd ..
+  echo "Install dependencies in Venv"
+  python3 -m venv .venv
+  source .venv/bin/activate
+  pip install -r tools/AutoTuner/requirements.txt
```

Comment:
`setup.sh` script I mentioned in the other PR would handle this part.

Reply:
Updated, please check.